DEMOQA/cypress/fixtures/testData.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "firstName": "Phuong",
+        "lastName": "Vo",
+        "userEmail": "vnphuong3001@gmail.com",
+        "invalidEmail": ["vnphuong3001","vnphuong3001@","vnphuong3001gmail.com"],
```

Comment:
For fixture file you can use like this:
```json
{
    "email": {
         "valid": #data,
         "invalid": #data
    }
}
```

Then access by `.email.valid`
It's just an option so you can choose what you think more suitable